router/chi/router.go

```diff
@@ -116,15 +116,32 @@ func (r chiRouter) registerKrakendEndpoints(endpoints []*config.EndpointConfig)
 			continue
 		}
 
-		r.registerKrakendEndpoint(c.Method, c.Endpoint, r.cfg.HandlerFactory(c, proxyStack), len(c.Backend))
+		r.registerKrakendEndpoint(c.Method, c, r.cfg.HandlerFactory(c, proxyStack), len(c.Backend))
 	}
 }
 
-func (r chiRouter) registerKrakendEndpoint(method, path string, handler http.HandlerFunc, totBackends int) {
+func (r chiRouter) registerKrakendEndpoint(method string, endpoint *config.EndpointConfig, handler http.HandlerFunc, totBackends int) {
 	method = strings.ToTitle(method)
+	path := endpoint.Endpoint
 	if method != http.MethodGet && totBackends > 1 {
-		r.cfg.Logger.Error(method, "endpoints must have a single backend! Ignoring", path)
-		return
+		if endpoint.ExtraConfig["github.com/devopsfaith/krakend/proxy"] == nil {
```

Comment:
I'd use the constant `proxy.Namespace`

Reply:
Ah, okay. Thanks, I didn't know that this constant exists.